Workbooks/Azure Update Management Center/Azure Update Management Center/Azure Update Management Center.workbook

```diff
@@ -0,0 +1,950 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "57df95b6-bf4e-43d7-bac0-22f2903ede45",
+            "version": "KqlParameterItem/1.0",
+            "name": "Subscription",
+            "type": 6,
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "typeSettings": {
+              "additionalResourceOptions": [],
+              "includeAll": false,
+              "showDefault": false
+            },
+            "timeContext": {
+              "durationMs": 86400000
+            }
+          },
+          {
+            "id": "05578175-fbe8-4dd2-9c6a-dec2f503d6cf",
+            "version": "KqlParameterItem/1.0",
+            "name": "Location",
+            "type": 8,
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "value": [
+              "value::all"
+            ],
+            "typeSettings": {
+              "additionalResourceOptions": [
+                "value::all"
+              ],
+              "includeAll": true
+            },
+            "defaultValue": "value::all"
+          },
+          {
+            "id": "3f095357-8b61-4bd5-bb16-bb03da10f44c",
+            "version": "KqlParameterItem/1.0",
+            "name": "ResourceType",
+            "type": 7,
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "value": [
+              "microsoft.hybridcompute/machines",
+              "microsoft.compute/virtualmachines"
+            ],
+            "typeSettings": {
+              "additionalResourceOptions": [],
+              "showDefault": false
+            },
+            "jsonData": " [\r\n     { \"value\": \"microsoft.compute/virtualmachines\", \"label\": \"Azure Virtual Machine\", \"selected\":true}, \r\n     \r\n     { \"value\": \"microsoft.hybridcompute/machines\", \"label\": \"Arc enabled server\", \"selected\":true }]"
+          },
+          {
+            "id": "2eeb36dc-a686-4ace-b90f-cfcf141e097b",
+            "version": "KqlParameterItem/1.0",
+            "name": "TimeRange",
+            "type": 4,
+            "isRequired": true,
+            "value": {
+              "durationMs": 2592000000
```

Comment:
This seems like a large default value, sometimes it could cause the workbook to return data slower, is there a lower timerange your workbook could use?